### PR TITLE
First pass at future events

### PR DESCRIPTION
**Pull request by pbuttigieg**
Relevant to #151 #156

### CODE REVIEW

File: book/thematics/events/graphs/futureEvent.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
+{
+    "@context": {
+        "@vocab": "https://schema.org/"
+    },
+    "@type": "Event",
+    "@id": "https://pidOfThisJsonLDdoc.org/",
+    "name": "The event name",
+    "EventStatus": "EventScheduled",
```
Comment (pbuttigieg) [inline book/thematics/events/graphs/futureEvent.json:8]: This property is what we would use (together with the start/endDate to express this is a future or planned event. Some concerns being clarified in https://github.com/schemaorg/schemaorg/issues/3236

**Comment by jmckenna** [comment]
List of mapped tables missing in pull request:

- [ ] vessel_equipment
- [ ] institution_affiliation
- [ ] cruise_track
- [ ] linked_projects
- [ ] collab_opportunities
- [ ] ocean_region

**Comment by erinmr** [comment]
@pbuttigieg - I am not sure if this is the right thread to add. In the Island Sampling Day example, we had both planned events and planned samples in GEOME. Is there a place for SampleStatus to indicate planned samples?

**Comment by pbuttigieg** [comment]
@erinmr yes, this is the right PR - see [this comment](https://github.com/iodepo/odis-arch/pull/158#discussion_r1048521217)

**Comment by erinmr** [comment]
@pbuttigieg - would you think the json-ld would come from GEOME or iSamples or should we think about creating it separately?

**Pull request merged**